pkg/cmd/init/options.go

```diff
@@ -272,8 +281,8 @@ func getPath(args []string) (string, error) {
 	}
 }
 
-// validatePath checks the path is valid or not.
-func validatePath(path string) error {
+// validateLocalTemplateRepoPath checks the path is valid or not.
+func validateLocalTemplateRepoPath(path string) error {
```

Comment:
`validatePath` works offline mode only, if miss understanding, `validateLocalPath` can be better

Reply:
it refers a local path, which should contain templates, but only validate it's a dir, not a file, or even not exists.
this function doesn't check whether templates are valid or not.

Reply:
yeah, you are right, so the suffix is "Path". "LocalTemplateRepoPath" asks the path refer to a local template repo, although the func only checks it's a dir or not for now, more validation for local template repo can be added in the future.